composer.json

```diff
@@ -176,6 +176,7 @@
         "drupal/redirect": "^1.0@alpha",
         "drupal/metatag": "^1.0",
         "drupal/google_tag": "1.x-dev",
-        "drupal/rabbit_hole": "1.0-beta3"
+        "drupal/rabbit_hole": "1.0-beta3",
+        "drupal/admin_toolbar": "1.x-dev"
```

Comment:
1. This module is mostly for internal use, for admin pages, and will not affect to front-end part, that's why make sense to use dev version to get new fixes from dev branch while updating modules without needs to update the version in the repository. 
2. ok, I will add it.